src/main/java/moomoo/command/NotificationCommand.java

```diff
@@ -29,11 +29,14 @@ public void execute(ScheduleList calendar, Budget budget, CategoryList catList,
                         Ui ui, Storage storage) throws MooMooException {
         String alert = "";
         if (expenditure > budget.getBudgetFromCategory(cat)) {
-            alert = "\u001B[36m" + "You have exceeded your budget of " + budget.getBudgetFromCategory(cat) + " for " + cat + "!\n";
+            alert = "\u001B[36m" + "You have exceeded your budget of " + budget.getBudgetFromCategory(cat)
+                    + " for " + cat + "!\n";
         } else if (expenditure == budget.getBudgetFromCategory(cat)) {
-            alert = "\u001B[36m" + "You have reached your budget limit of " + budget.getBudgetFromCategory(cat) + ".\n";
+            alert = "\u001B[36m" + "You have reached your budget limit of " + budget.getBudgetFromCategory(cat)
+                    + ".\n";
         } else if (expenditure > budget.getBudgetFromCategory(cat) * 0.9) {
-            alert = "\u001B[36m" + "You are reaching your budget limit of " + budget.getBudgetFromCategory(cat) + " for " + cat
+            alert = "\u001B[36m" + "You are reaching your budget limit of " + budget.getBudgetFromCategory(cat)
+                    + " for " + cat
```

Comment:
could change the colour for each type of alert. eg exceed limit -> red, reach limit -> orange, reaching -> yellow